### PR TITLE
Sync genconfig.py output file extensions with rest of repo

### PR DESCRIPTION
**Pull request by siculars**
Change output extention from ".yaml" to ".yml". Help output, README and default config file extensions all use ".yml".

### CODE REVIEW

File: genconfig.py [hunk 1]
```diff
@@ -32,8 +32,8 @@ def dump_yaml(directory, filename, servers, port):
     arguments = parser.parse_args()
 
     if arguments.scylla:
-        dump_yaml(arguments.directory, 'scylla_servers.yaml', arguments.servers, scylla_port)
+        dump_yaml(arguments.directory, 'scylla_servers.yml', arguments.servers, scylla_port)
```
Comment (tzach) [inline genconfig.py:35]: @glommer can you please review?
How did this work so far? file name under prometheus dir are already with `yml` suffix

**Comment by tzach** [comment]
@glommer ping

**Comment by glommer** [comment]
Hi.

I am ok with the change. genconfig generates the files in a different
location than the standard files, and the files have to be passed
explicitly with the -n and -s switches to start-all.

Because of that the extension doesn't matter - they will be bind mounted to
the correct locatiom regardless of their name.

But consistency is good so I am ok with it

On Jul 30, 2017 3:25 AM, "Tzach Livyatan" <notifications@github.com> wrote:

@glommer <https://github.com/glommer> ping

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/scylladb/scylla-grafana-monitoring/pull/203#issuecomment-318884232>,
or mute the thread

**Comment by tzach** [comment]
Thanks @siculars @glommer

**Pull request merged**